src/styles/_layout.scss

```diff
@@ -60,6 +60,9 @@
   }
 
   > main:not(.scrolling) > article > div {
+    /* fix: dirty overlapping of orange outline with neighboring rows */
+    margin: $engine_hover_outline_width 0;
```

Comment:
as discussed before, in the end we want to switch to a border (transparent vs color) approach for highlighting. requires #4 

I will merge it for now but we have to keep that in mind

Reply:
Yes, we can try this in the next iteration.